app/src/main/java/org/example/uiController/UIController.java

```diff
@@ -133,13 +138,13 @@ private void handleRunCommand(String[] command){
             Job userSubmittedJob = new Job(jobName, jobPriority, jobTime, currentDate);
 
             if (schedulerThread == null){
-                schedulerThread = this.startThread("Scheduler", scheduler);
+                schedulerThread = this.startThread(scheduler);
             }
 
             this.scheduler.addJob(userSubmittedJob);
             System.out.println("Job '" + jobName + "' added to the queue.");
             if (enableDispatcher && dispatcherThread == null) {
-                dispatcherThread = this.startThread("Dispatcher", dispatcher);
```

Comment:
The log for the name was removed from `startThread` so this param is no longer being used.